### PR TITLE
GFORMS-2093 - Create a new destination to send gForm data to the new …

### PR DESCRIPTION
**Pull request by DursunGundogan**
…data store

### CODE REVIEW

File: app/uk/gov/hmrc/gform/sdes/SdesConnector.scala [hunk 1]
```diff
@@ -33,7 +33,7 @@ class SdesConnector(wSHttp: WSHttp, sdesBaseUrl: String, sdesBasePath: String, h
     HttpReadsInstances.throwOnFailure(HttpReadsInstances.readEitherOf(HttpReadsInstances.readRaw))
 
   def notifySDES(payload: SdesNotifyRequest)(implicit hc: HeaderCarrier): Future[HttpResponse] = {
-    logger.debug(s"SDES notification request: ${Json.stringify(Json.toJson(payload))}")
+    logger.info(s"SDES notification request: ${Json.stringify(Json.toJson(payload))}")
```
Comment (lxol) [inline app/uk/gov/hmrc/gform/sdes/SdesConnector.scala:36]: does SdesNotifiyRequest contains personal information? We can see `info` logs in prod kibana.

Comment (DursunGundogan) [reply app/uk/gov/hmrc/gform/sdes/SdesConnector.scala:36]: It does not contain personal information. We should see the notification logs in kibana.

Comment (lxol) [reply app/uk/gov/hmrc/gform/sdes/SdesConnector.scala:36]: It looks like it contains a file name. Is that any name ? Can it be like "JohnDoe_TaxReturns_2023.pdf" ?

Comment (DursunGundogan) [reply app/uk/gov/hmrc/gform/sdes/SdesConnector.scala:36]: No, it looks like ``envelopeId`.zip`. The file name does not contain any personal information.

**Review by lxol — APPROVED** [review]

**Pull request merged**